nautobot/extras/tests/test_api.py

```diff
@@ -2631,6 +2661,192 @@ def test_populate_relationship_associations_on_site_create(self):
             ).exists()
         )
 
+    def test_create_vlan_with_required_device_destinations_m2m(self):
+        """
+        Test that we can't create a vlan without specifying at least one device for this required relationship
+        1. Try creating a vlan when no device exists
+        2. Try creating a vlan with no specified device data in the list of required objects
+        3. Try creating a vlan when all required data is present
+        """
+        self.user.is_superuser = True
+        self.user.save()
+
+        create_data = {
+            "vid": 1,
+            "name": "New VLAN",
+            "status": "active",
+        }
+
+        # Try creating a vlan when no device exists
+        response = self.client.post(
+            reverse(get_route_for_model(VLAN, "list", api=True)),
+            data=create_data,
+            format="json",
+            **self.header,
+        )
+        expected_error_json = [
+            {
+                "cr_vlans-devices__destination": "Vlans require at least one device, but no devices exist yet. "
```

Comment:
Why `cr_vlans-devices__destination` here but `vlans-devices` in the next expected error? Seems inconsistent to me.